Xamarin.Forms.Core/View.cs

```diff
@@ -49,7 +49,9 @@ protected internal View()
 						foreach (IElement item in _gestureRecognizers.OfType<IElement>())
 							item.Parent = this;
 						break;
-				}
+                    case NotifyCollectionChangedAction.Move:
```

Comment:
`GestureRecognizers` is exposed as a `IList`, so a `Move` is very unlikely to happen.